serverless/knative_eventing/serverless-subscriptions.adoc

```diff
@@ -6,6 +6,57 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-Subscriptions deliver events to event sinks from a Channel.
+You can create a subscription to enable event delivery from a channel to a service or other event sink.
 
-include::modules/serverless-creating-subscriptions.adoc[leveloffset=+1]
+[IMPORTANT]
+====
+Knative Eventing is a Technology Preview feature. The InMemoryChannel type is provided for development use only, and should not be used in a production environment.
```

Comment:
We have a `technology-preview.adoc` file that we include so that we use consistent wording for TP features. Suggest following that here (just search the repo for that file and you'll see how it's used).

Reply:
I actually use it somewhere else, but this note is specifically to call out the thing about InMemoryChannel rather than the TP status. I'll remove the first line to make this the focus. Thanks.